tests/modules/images/image-loading-optimization/storage/post-type-tests.php

```diff
@@ -138,22 +148,11 @@ public function test_ilo_store_url_metric() {
 
 		$url_metrics = ilo_parse_stored_url_metrics( $post );
 		$this->assertCount( 1, $url_metrics );
-		$this->assertArrayHasKey( 'timestamp', $url_metrics[0] );
-		$this->assertIsFloat( $url_metrics[0]['timestamp'] );
-		$this->assertLessThanOrEqual( microtime( true ), $url_metrics[0]['timestamp'] );
```

Comment:
Irrelevant to test now that the `timestamp` is required at the time of instantiation of `ILO_URL_Metric`. Previously it would get added by `ilo_unshift_url_metrics()`.